### PR TITLE
Added Lines

### PR DESCRIPTION
**Pull request by UgoAsika**
Changed.

### CODE REVIEW

**Comment by UgoAsika** [comment]
Can you remove one L?

File: README.md [hunk 1]
```diff
@@ -10,5 +10,5 @@ I want to add more features into the bot, these are my goals from most to least
 <ol start="1">
  <li>More responses</li>
  <li>More personality systems</li>
- <li>Dynamicly changing personality based on the flow of the conversation</li>
+ <li>Dynamicly ll changing personality based on the flow of the conversation</li>
```
Comment (UgoAsika) [inline README.md]: Can you remove one L?

Comment (UgoAsika) [reply README.md]: Add an Extra L after based please.

File: README.md [hunk 2]
```diff
@@ -10,5 +10,5 @@ I want to add more features into the bot, these are my goals from most to least
 <ol start="1">
```
Comment (UgoAsika) [inline README.md:10]: Fix the s please.

**Review by UgoAsika — COMMENTED** [review]
Approved.

File: README.md (hunk 1 above)
Comment (MeName13) [inline README.md]: I noticed an extra "l" here

**Review by MeName13 — CHANGES_REQUESTED** [review]
Extra L should be deleted

**Review by MeName13 — APPROVED** [review]
looks good!

**Pull request merged**